### ISSUE TITLE
Monitoring/Ranging not working on MotoZ

### ISSUE THREAD

**Issue by eugenelozada** [Estimote/Android-SDK#199]
Ranging and monitoring listeners don't seem to firing when testing on a Moto Z.   Eddystone listeners seem to be ok though.

**Comment by pawelDylag** [comment]
Hello @eugenelozada 

This may be related to a problem with hardware filtering on some phones. [Here is one possible solution to your problem](https://github.com/Estimote/Android-SDK/issues/197#issuecomment-285326382)
Let me know it that helps :)

**Comment by eugenelozada** [comment]
Will try this as soon as I get back to my beacons and will send an update.  Thanks for the response!

**Comment by eugenelozada** [comment]
Ok monitoring and ranging are firing now.  But a couple of things I noticed:

I started getting these on app start up after I disabled hardware filtering:
`com.estimote.sdk.service.BeaconService$7.onLeScan:1016 Nearable packet parsing error`

Second, I have 6 beacons next to me with the same UUID and the region filter set to only look at the UUID.  I expected most of if not all of them to be listed in the onEnteredRegion listener, but I seem to be getting just one of them. Succeeding scans (every minute or so) would then trigger an exit/enter with a different beacon listed, but always just one beacon at a time.  Is this expected behavior?

**Comment by pawelDylag** [comment]
Hello @eugenelozada 

Please consider switching to our fresh sdk version 1.0.0 ([See the migration guide](https://github.com/Estimote/Android-SDK/blob/master/Docs/switching_to_1.0.0.md))
Does this problem still occur?

**Comment by eugenelozada** [comment]
OK, updated to the new SDK and some issues encountered.  I'm not sure if I'm missing anything but here are my results:

I have 6 devices next to me, but I'm still seeing just one:

` (OnEnteredRegion) : [Beacon{macAddress=[F1:FB:19:E2:74:58], proximityUUID=b9407f30-f5f8-466e-aff9-25556b57fe6d, major=33006, minor=8372, measuredPower=-63, rssi=-75}]`

Other issues:
- Still need to disable hardware filtering
- When I start monitoring, the app now crashes if I don't initialize the SDK with EstimoteSdk.initialize(context, appId, appToken).  Previously it didn't require me to do this
- Now when I run the app, I get one monitoring result, but also get several 403s, so I think I'm not reaching the Estimote cloud for some reason:
' at java.lang.Thread.run(Thread.java:761)
03-30 14:57:04.882 22554-22554/com.accenture.lqs W/System.err: com.estimote.coresdk.repackaged.retrofit_v1_9_0.retrofit.RetrofitError: 403 Forbidden'

**Comment by pawelDylag** [comment]
Hey @eugenelozada !

Thanks for the info! We will work a bit with those issues with hardware filtering. The initialization is needed for analytics to work - you need to disable it to work without initialize(...)
As for the "403" errors - are you sure you initialized our SDK with the proper appId and appToken?
Are you using `SecureBeaconRegion`, or just `BeaconRegion`?
Also, I found out that some unnecessary logs were not disabled before release. They come from underlying `resolvers` which are trying to resolve in cloud not owned beacons around you. Could you verify, if there are other beacons around you that are not yours?  if yes, we just need to disable this log, because it is not important to you :)

Have a nice day!

**Comment by eugenelozada** [comment]
- I have the following lines in my onCreate()

'EstimoteSDK.enableRangingAnalytics(false);
 EstimoteSDK.enableMonitoringAnalytics(false);
 EstimoteSDK.enableGpsPositioningForAnalytics(false);'

Before my call to `beaconManager.connect()` But it's still hard crashing if I don't initialize the SDK first.  Should I be disabling analytics in a different manner?

- As for the 403s, yeah I set the appId and token from the cloud console when i create the app.  and I'm still getting those errors.  Also, I'm using `BeaconRegion`  to set the Region filter.

- As for the actual monitoring, I'll also add that I'm  getting one onRegionEntered() callback with that one beacon listed . In the previous version of the SDK, I'll periodically get other onRegionEntered() callbacks where the different beacons I are individually returned per invocation of the callback

- I can confirm that the only beacons around me are the 6 I have.  I tested this with the official Estimote app and those are the only 6 that show up
